test/plugin/test_in_rds_slowlog.rb

```diff
@@ -87,6 +88,7 @@ def setup
     username test_rds_user
     password test_rds_password
     interval 0
+    backup_table mysql.slow_log_custom_backup
```

Comment:
I think it is better not to add `backup_table` to the default setting.
Would you like to add `backup_table` in `test_backup`?

```ruby
d = create_driver(CONFIG + 'backup_table mysql.slow_log_custom_backup')
```